posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -607,43 +607,59 @@ public class PostHogReplayIntegration(
         view: View,
         maskableWidgets: MutableList<Rect>,
     ) {
-        try {
-            val semanticsOwner =
-                (view as? RootForTest)?.semanticsOwner ?: run {
-                    config.logger.log("View is not a RootForTest: $view")
-                    return
-                }
-            val semanticsNodes = semanticsOwner.getAllSemanticsNodes(true)
+        val latch = CountDownLatch(1)
```

Comment:
given that this method is only executed when `view.isComposeView()`, its fine to do thread jumping since its a requirement for compose anyway.